scripts/generate_landings_risk_report.py

```diff
@@ -1795,9 +1795,11 @@ def calculate_maintenance_effectiveness(
 
             def format_maintenance_effectiveness(period: relativedelta) -> str:
                 me = calculate_maintenance_effectiveness(period)
-                return "ME: {}%, WeightedBurnDownTime: {} y".format(
-                    round(me["ME"], 2),
-                    round(me["WBDTime"], 2),
+                return "ME: {}%, WeightedBurnDownTime: {} y\nOpened bugs: {}\nClosed bugs: {}".format(
```

Comment:
nit: I'm not sure how this appears in the UI, but I think the URLs may be quite long. Can we use an MD link instead?

```suggestion
                return "ME: {}%, WeightedBurnDownTime: {} y\nOpened bugs: [Bugzilla query]({})\nClosed bugs: [Bugzilla query]({})".format(
```

Reply:
Good idea, done in 919bc66.